cljfmt/src/cljfmt/main.clj

```diff
@@ -156,7 +156,7 @@
       (assoc :indents (merge (:indents default-options)
                              (:indents options {})))))
 
-(def default-paths ["src" "test" "project.clj"])
+(def default-paths ["src" "test" "project.clj" "deps.edn"])
```

Comment:
Let's keep this PR focused and remove `"deps.edn"` for now. That should be part of a separate PR.

Reply:
Probably fine to just feed it in manually when desired; it's not quote-unquote "code" in the same sense as `project.clj`.